pkg/services/factory.go

```diff
@@ -104,22 +105,14 @@ func (f *defaultFactory) GetKubeService() (kube.Kube, error) {
 }
 
 func (f *defaultFactory) GetGitClients(ctx context.Context, gpClient gitproviders.Client, params GitConfigParams) (git.Git, gitproviders.GitProvider, error) {
-	isExternalConfig := models.IsExternalConfigRepo(params.ConfigRepo)
-
-	var providerUrl string
-
-	switch {
-	case !params.IsHelmRepository:
-		providerUrl = params.URL
-	case isExternalConfig:
-		providerUrl = params.ConfigRepo
-	default:
-		return nil, nil, nil
```

Comment:
I am not really sure when we will ever hit this default option. I believe with the new directory structure we will always have a config-repo. I also deleted the two tests associated with this.